src/Data/Voting.hs

```diff
@@ -62,10 +62,18 @@ instance Eq v => Category (DCat v) where
 data DelegationMap = DelegationMap {
         _delegationMap
             :: Map
-                (Voter, Topic) -- who delegates in which topic
-                Voter -- to whom delegates
+                Voter -- who delegates in which topic
+                (Map Topic Voter) -- to whom delegates
```

Comment:
comments need update here.  this great evidence that comments should *never* carry information, because it'll be *always* outdated.  (-: